ui/app/src/components/ComponentMenu.tsx

```diff
@@ -216,6 +223,15 @@ export default function ComponentMenu(props: ComponentMenuProps) {
             />
           </MenuItem>
         }
+        {
+          (item && (item.state === 5 || item.state === 6 || item.state === 2)) &&
```

Comment:
I don't think the state will ever be a plain number. This is supposed to be a bit map.